lib/block-editor-settings.php

```diff
@@ -89,6 +89,23 @@ function gutenberg_get_block_editor_settings( $settings ) {
 		);
 	}
 
+	$settings['typesNotSupportedByServer'] = array();
+
+	// Check if WebP images can be edited.
+	if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
+		$settings['typesNotSupportedByServer']['image/webp'] = true;
+	}
+
+	// Check if AVIF images can be edited.
+	if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/avif' ) ) ) {
+		$settings['typesNotSupportedByServer']['image/avif'] = true;
+	}
+
+	// Check if HEIC images can be edited.
+	if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/heic' ) ) ) {
+		$settings['typesNotSupportedByServer']['image/heic'] = true;
+	}
```

Comment:
The release-specific code needs to go into the `compat/wordpress-6.8` (next major version) folder. This will probably be a part of the `get_default_block_editor_settings` function during backport.

Reply:
Ah, ok thanks for clarifying. I will move

Reply:
moved in https://github.com/WordPress/gutenberg/pull/68644/commits/f97809b714afbed01d7980d6a21d0efb8ea086d6 - is this what you were thinking?